src/mame/dec/microvax3100.cpp

```diff
@@ -0,0 +1,170 @@
+  |       | C  |     |   |                     __________  __________  __________  ___________       |_____________| |
+  |       | O  |     | C |                    |74F245PC_| |74F32NQST| |74F32NQST| |_74F543N__|        ______________ |
+  |       | N  |     | O |                     __________  __________  __________  ___________       | EPROM       | |
+  |       | N  |     | N |                    |_74F244N_| |SN74LS09N| |74F00NQST| |74HCT245N_|       |_____________| |
+  |       |    |     | N |                     __________  __________  __________  ___________        ___________    |
+  |       |    |     |   |                    |MC74F240N| |74F32NQST| |_________| |_74F543N__|       |_74LS244N_|    |
+  |       |    |     |   |                                                         ___________        ___________    |
+  |       |    |     |___|       ______________________    ______________________ |74HCT245N_|       |_74F374N__|    |
+  |       |____|                | :::::::::::::::::::: |  | ::::::::::::::::::: |  ___________                       |
+  |                                                                               |_74F543N__|                       |
+  |__________________________________________________________________________________________________________________|
+
+***********************************************************************************************************************/
+
+#include "emu.h"
+#include "cpu/t11/t11.h"
+
+#include "machine/am79c90.h"
+//#include "machine/ncr5380.h"
+//#include "machine/scnxx562.h"
+#include "machine/terminal.h"
+
+
+namespace {
+
+class microvax3100_state : public driver_device
+{
+public:
+	microvax3100_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+		, m_terminal(*this, "terminal")
+	{ }
+
+	void microvax3100(machine_config &config);
+
+private:
+	required_device<t11_device> m_maincpu;
+	required_device<generic_terminal_device> m_terminal;
+};
+
+
+// Input ports
+static INPUT_PORTS_START( microvax3100 )
+INPUT_PORTS_END
+
+
+void microvax3100_state::microvax3100(machine_config &config)
+{
+	// Basic machine hardware
+	T11(config, m_maincpu, 66.6667_MHz_XTAL); // Need proper CPU here
+
+	AM7990(config, "lance1", 0); // AMD AM7990PC/80
+
+	// NCR5380(...)
+
+	// DUSCC68562(...) // Signetics SCN68562C4N48
+
+	// Video hardware
+	GENERIC_TERMINAL(config, m_terminal, 0);
+}
+
+ROM_START( mvax3100 )
+	ROM_REGION( 0x40000, "maincpu", ROMREGION_ERASEFF )
+	ROMX_LOAD( "dec89_23-116e8-00_system_rom_lo_word.e24", 0x00000, 0x20000, CRC(69ef8cf5) SHA1(a59a500921278dc356a519cb435641426c844779), ROM_NIBBLE | ROM_SHIFT_NIBBLE_LO | ROM_SKIP(1) )
+	ROMX_LOAD( "dec89_23-117e8-00_system_rom_hi_word.e25", 0x00000, 0x20000, CRC(df572ac3) SHA1(5e91d0f4fc8442e3ebc2424d9f85078ba00d2de5), ROM_NIBBLE | ROM_SHIFT_NIBBLE_HI | ROM_SKIP(1) )
```

Comment:
I don't think this loading is correct. These are supposed to be 65536x16 EPROMs, not 131072x4.

Reply:
Fixed!